control/preimage/src/main.rs

```diff
@@ -241,6 +260,13 @@ async fn run() -> Result<(), Box<dyn std::error::Error>> {
                 send_xcm_asset_hub(&context, vec![force_xcm_version(), set_ethereum_fee]).await?;
             utility_force_batch(vec![call1, call2])
         }
+        Command::UpdateAsset(params) => {
```

Comment:
Rather structure the XCM this way, to avoid having to make two separate XCM calls to AssetHub

```
SendXCM(AssetHub,
  Xcm([
    Transact(MakeSufficient)
    Transact(SetMetadata)
  ])
)
```

As an example see the Initialize command, where we perform multiple Transacts in a single downward XCM.